client-data/tools/eraser/eraser.js

```diff
@@ -39,6 +39,22 @@
 		"type": "delete",
 		"id": ""
 	};
+
+	function inDrawingArea(elem) {
+		if (window.navigator.userAgent.indexOf("MSIE") === -1) {
```

Comment:
```suggestion
		if (Tools.drawingArea.contains) {
```

we don't need UA sniffing